tasks.py

```diff
@@ -517,67 +519,147 @@ def test_api(c, product, backend=''):
-      'lua2',
-      'lua2-dnssec'
-    ],
-    tinydns = [
-      'tinydns'
+        'geoip',
+        'geoip-nsec3-narrow'
+        # FIXME: also run this with the mmdb we ship
     ],
+    lua2 = ['lua2', 'lua2-dnssec'],
+    tinydns = ['tinydns'],
     remote = [
-      'remotebackend-pipe',
-      'remotebackend-unix',
-      'remotebackend-http',
-      'remotebackend-zeromq',
-      'remotebackend-pipe-dnssec',
-      'remotebackend-unix-dnssec',
-      'remotebackend-http-dnssec',
-      'remotebackend-zeromq-dnssec'
+        'remotebackend-pipe',
+        'remotebackend-unix',
+        'remotebackend-http',
+        'remotebackend-zeromq',
+        'remotebackend-pipe-dnssec',
+        'remotebackend-unix-dnssec',
+        'remotebackend-http-dnssec',
+        'remotebackend-zeromq-dnssec'
     ],
     lmdb = [
-      'lmdb-nodnssec-both',
-      'lmdb-both',
-      'lmdb-nsec3-both',
-      'lmdb-nsec3-optout-both',
-      'lmdb-nsec3-narrow'
+        'lmdb-nodnssec-both',
+        'lmdb-both',
+        'lmdb-nsec3-both',
+        'lmdb-nsec3-optout-both',
+        'lmdb-nsec3-narrow'
+    ],
+    gmysql = [
+        'gmysql',
+        'gmysql-nodnssec-both',
+        'gmysql-nsec3-both',
+        'gmysql-nsec3-optout-both',
+        'gmysql-nsec3-narrow',
+        'gmysql_sp-both'
+    ],
+    gpgsql = [
+        'gpgsql',
+        'gpgsql-nodnssec-both',
+        'gpgsql-nsec3-both',
+        'gpgsql-nsec3-optout-both',
+        'gpgsql-nsec3-narrow',
+        'gpgsql_sp-both'
     ],
-    gmysql   = ['gmysql',     'gmysql-nodnssec-both',   'gmysql-nsec3-both',   'gmysql-nsec3-optout-both',   'gmysql-nsec3-narrow',   'gmysql_sp-both'],
-    gpgsql   = ['gpgsql',     'gpgsql-nodnssec-both',   'gpgsql-nsec3-both',   'gpgsql-nsec3-optout-both',   'gpgsql-nsec3-narrow',   'gpgsql_sp-both'],
-    gsqlite3 = ['gsqlite3', 'gsqlite3-nodnssec-both', 'gsqlite3-nsec3-both', 'gsqlite3-nsec3-optout-both', 'gsqlite3-nsec3-narrow'],
+    gsqlite3 = [
+        'gsqlite3',
+        'gsqlite3-nodnssec-both',
+        'gsqlite3-nsec3-both',
+        'gsqlite3-nsec3-optout-both',
+        'gsqlite3-nsec3-narrow'
+    ],
+    godbc_sqlite3 = ['godbc_sqlite3-nodnssec'],
+    godbc_mssql = [
+        'godbc_mssql',
+        'godbc_mssql-nodnssec',
+        'godbc_mssql-nsec3',
+        'godbc_mssql-nsec3-optout',
+        'godbc_mssql-nsec3-narrow'
+    ],
+)
+
+godbc_mssql_credentials = {"username": "sa", "password": "SAsa12%%"}
+
+godbc_config = (
+    "[pdns-mssql-docker]\n"
+    "Driver=FreeTDS\n"
+    "Trace=No\n"
+    "Server=127.0.0.1\n"
+    "Port=1433\n"
+    "Database=pdns\n"
+    "TDS_Version=7.1\n"
+    "\n"
+    "[pdns-mssql-docker-nodb]\n"
+    "Driver=FreeTDS\n"
+    "Trace=No\n"
+    "Server=127.0.0.1\n"
+    "Port=1433\n"
+    "TDS_Version=7.1\n"
+    "\n"
+    "[pdns-sqlite3-1]\n"
+    "Driver = SQLite3\n"
+    "Database = pdns.sqlite3\n"
+    "\n"
+    "[pdns-sqlite3-2]\n"
+    "Driver = SQLite3\n"
+    "Database = pdns.sqlite32\n"
```

Comment:
Perhaps use a [triple quoted string](https://docs.python.org/3/library/stdtypes.html?highlight=triple%20quoted%20string#text-sequence-type-str) here?

Reply:
how about this one?

Reply:
Yes, I noticed that one an hour ago. Running tests against my fork. I will update this PR once confirmed I do not break anything